### ISSUE TITLE
Docker Hub CLI

### ISSUE THREAD

**Issue by olegburov** [docker/roadmap#117]
**Tell us about your request**
Docker's official command-line tool that makes `docker` easier to use with [Docker Hub](https://hub.docker.com/).

**Which service(s) is this request for?**
Docker Hub

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
Some people prefer to use GUI, while others like terminal. 

I would like to have an easier and more seamless way to work with Docker Hub from the command line. Create repositories, tags, teams, adding collaborators, team's members and managing your organization without leaving the command line. Seamless experience to bring Docker Hub in Continuous Integration (CI) pipeline.

**Are you currently working around the issue?**
Reverse engineering existing APIs from a browser.

**Additional context**
Something like https://cli.github.com/

**Comment by jaythamke** [comment]
Hi @olegburov I am looking for Personal Access Token rotation scenario using docker-hub cli. I came to know from `slack > general` channel that managing access token will be covered. I would like to know if output format in JSON is available. Thanks

**Comment by olegburov** [comment]
@silvin-lubecki can you please help me here 👆?

**Comment by silvin-lubecki** [comment]
Hello @jaythamke , yes the hub tool CLI has an option to output all commands as json. You will be able to create, list and remove all your personal access tokens with it 👍

**Comment by afmsavage** [comment]
Is there a way to set an org as the default place to list repos and tags? Right now it seems I can only list repos for my own account and not orgs I am apart of.

**Comment by silvin-lubecki** [comment]
@afmsavage actually the `tag ls` command takes an organization as optional argument:
```
$ hub-tool repo ls --help
List all the repositories from your account or an organization

Usage:
  hub-tool repo ls [ORGANIZATION]
...
```
So `hub-tool repo ls myorg` will show all the organization repositories. Does that fit with your needs?

**Comment by afmsavage** [comment]
Yes definitely.  I'm sorry I completely missed that you could add the Org to that command @silvin-lubecki .  Thank you for following up on this.
